### PR TITLE
Avoid NPE in case if resource is not found

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Avoid NPE in case if resource is not found

### What issues does this PR fix or reference?
Avoid NPE in case if resource is not found
#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-build

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Review by tolusha — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

File: selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/WorkspaceDtoDeserializer.java [hunk 1]
```diff
@@ -31,32 +33,31 @@
...
+              WorkspaceDtoDeserializer.class,
+              format("/templates/workspace/%s/%s", infrastructure, templateName));
+      return DtoFactory.getInstance()
+          .createDtoFromJson(Resources.toString(url, Charsets.UTF_8), WorkspaceConfigDto.class);
+    } catch (IOException | IllegalArgumentException | JsonSyntaxException e) {
+      LOG.error(
+          "Fail to read workspace template {} for infrastructure {} because {} ",
+          templateName,
+          infrastructure,
+          e.getLocalizedMessage());
```
Comment (sleshchenko) [inline selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/WorkspaceDtoDeserializer.java]: Are you sure that you need localized message here?

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Review by vparfonov — APPROVED** [review]

**Review by artaleks9 — APPROVED** [review]

**Review by tolusha — APPROVED** [review]

**Pull request merged**